client/v3/kubernetes/interface.go

```diff
@@ -36,8 +36,9 @@ type Interface interface {
 	// If opts.Revision is non-zero, the key-value pairs are retrieved at the specified revision.
 	// If the required revision has been compacted, the request will fail with ErrCompacted.
 	// If opts.Limit is greater than zero, the number of returned key-value pairs is bounded by the limit.
-	// If opts.Continue is not empty, the listing will start from the key immediately after the one specified by Continue.
-	// The Continue value should be the last key returned in a previous paginated ListResponse.
+	// If opts.Continue is not empty, the listing will start from the key specified by it.
+	// The Continue value should be the last key returned in a previous
```

Comment:
When Paginating the Continue should be set to last observed key with "\x00" appended to it.